### ISSUE TITLE
escalation chain didnt run

### ISSUE THREAD

**Issue by paneda1998** [grafana/oncall#1187]
Hi i have some critical problems
after upgading from 1.1.8 to 1.1.16

there are some problems with alert groups it seems escalation chains didnt run anyway
more details:
i see this for all alerts:
![image](https://user-images.githubusercontent.com/52179148/213869860-e2f769de-21e5-47a3-8691-84b6e7754e54.png)

**Comment by paneda1998** [comment]
before updating i didnt have any problem and also i didnt change anything (code, escalation chains,routes ,...)

**Comment by Matvey-Kuk** [comment]
@paneda1998 do you have any more details? Logs, anything would be helpful

**Comment by paneda1998** [comment]
Hey
I did more detailed checks after checking the logs of engine celery rabbit and mysql but I didn't notice any kind of error.
But by checking the resource consumption of the containers, I noticed that there was a noticeable increase in CPU consumption, which I think is mostly related to MySQL.
after increasing the resource it seems everything is ok.
and i didnt change anything at all just upgrading from 1.1.8 to 1.1.16
